### ISSUE TITLE
Cannot install

### ISSUE THREAD

**Issue by loranger** [mojtabaahn/laravel-web-logs#10]
Hello,

I try to install this package using my docker setup, but unfortunately I failed

```bash
$  project [logs] docker-compose run --rm php composer require mojtabaahn/laravel-web-logs
Creating project_composer_run ... done
Using version ^1.3 for mojtabaahn/laravel-web-logs
./composer.json has been updated
Running composer update mojtabaahn/laravel-web-logs
Loading composer repositories with package information
Updating dependencies
Lock file operations: 1 install, 0 updates, 0 removals
  - Locking mojtabaahn/laravel-web-logs (1.3.1)
Writing lock file
Installing dependencies from lock file (including require-dev)
Nothing to install, update or remove
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

In Router.php line 375:

  Argument 1 passed to Illuminate\Routing\Router::group() must be of the type
   array, null given, called in /var/www/app/vendor/mojtabaahn/laravel-web-lo
  gs/src/Providers/LaravelWebLogsServiceProvider.php on line 36


Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

I use composer 2 with PHP 7.4

```
$  project [logs] docker-compose run --rm php composer -V -vvv
Creating project_composer_run ... done
Reading ./composer.json
Loading config file ./composer.json
Checked CA file /etc/ssl/certs/ca-certificates.crt: valid
Executing command (/var/www/app): git branch -a --no-color --no-abbrev -v
Executing command (/var/www/app): git describe --exact-match --tags
Executing command (CWD): git --version
Executing command (/var/www/app): git log --pretty="%H" -n1 HEAD
Executing command (/var/www/app): hg branch
Executing command (/var/www/app): fossil branch list
Executing command (/var/www/app): fossil tag list
Executing command (/var/www/app): svn info --xml
Failed to initialize global composer: Composer could not find the config file: /tmp/composer/composer.json
To initialize a project, please create a composer.json file as described in the https://getcomposer.org/ "Getting Started" section
Reading /var/www/app/vendor/composer/installed.json
Running 2.0.7 (2020-11-13 17:31:06) with PHP 7.4.16 on Linux / 4.19.0-16-amd64
Composer version 2.0.7 2020-11-13 17:31:06
```

What did I miss ?

**Comment by mojtabaahn** [comment]
Hello there

We had same issue #4 and unfortunately there were not enough data to figure this out.

Can you confirm your using Laravel, not Lumen?

**Comment by loranger** [comment]
I do confirm I'm using Laravel.
I'll try to pack a fresh project in order to being able to reproduce the issue. Maybe it could help

**Comment by mojtabaahn** [comment]
Aright I believe the problem was registering routes in service provider's register method (instead of boot).

It now should be fixed in v1.3.2 that just been pushed. please update and check if it is solved.
